### ISSUE TITLE
Blocks are not harvestable in Survival

### ISSUE THREAD

**Issue by goldenapple3** [woodiertexas/architecture-extensions#9]
### Describe the bug

Building blocks don't have the "mineable/pickaxe", "mineable/axe" etc tags, so they take a long time to break and do not drop themselves when mined.

### Which side have you encountered the bug on?

Both

### Steps to reproduce

Place block and try to break it in Survival lol

### Additional context

```
architecture_extensions-0.3.4+1.18.2.jar
qsl-1.1.0-beta.19_qfapi-1.0.0-beta.22_fapi-0.56.0_mc-1.18.2.jar
```

**Comment by woodiertexas** [comment]
This should be fixed in the 0.4.1 release of Arch-Ex (https://github.com/woodiertexas/architecture-extensions/commit/59a09c5db0e4be6b7e8157c129bc32159d0925e5). I'll merge the changes done in 0.4.1 into the 1.18 version tomorrow.

**Comment by woodiertexas** [comment]
This issue has been fixed in v0.4.1 for both 1.19 and 1.18.2. You can get v0.4.1 for either version with the following links:

https://modrinth.com/mod/arch-ex/version/v0.4.1%2B1.19 - 1.19
https://modrinth.com/mod/arch-ex/version/v0.4.1%2B1.18.2 - 1.18.2

If you find any new bugs, feel free to make another bug report.
